lib/utils.py

```diff
@@ -53,7 +53,9 @@ def get_driverpath():
     tmprinter = TMPrinter()
     drivers = [str(x.absolute()) for x in Path('.').rglob('chromedriver*')]
     if not drivers:
-        drivers = shutil.which("chromedriver")
+        driver = shutil.which("chromedriver")
+        if driver is not None:
```

Comment:
It's nothing but can you just replace the `if driver is not None` by `if driver` ? I find it cleaner.

Reply:
The way you fixed my commit is fine. Thx for solving the problem!